test/redeemer/Redeemer.sol

```diff
@@ -119,15 +122,19 @@ contract Redeemer {
   /// @param p value of a specific principal according to the Illuminate Principals Enum
   /// @param u underlying token being redeemed
   /// @param m maturity of the market being redeemed
-  /// @param d sense wut (divider?) ?
-  /// @param a sense wut (adapter?) ?
+  /// @param d sense contract that splits the loan's prinicpal and yield
+  /// @param a sense adapter contract that controls the underlying token
   function redeem(uint8 p, address u, uint256 m, address d, address a) public returns (bool) {
```

Comment:
oof `adapter`.... have we officially adopted that term into our language? wasn't it kind a julian invention?

or, does the principal use that in their language? i'd think it'd be better to have a clearer explaination like `d` has